bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -373,23 +373,19 @@ class HasProps extends Backbone.Model
         throw new Error("models must be owned by only a single document")
       else
         @document._notify_attach(@)
```

Comment:
Saying this, I think, we may need to revisit this logic in general in future, but for now this PR fixes the performance issue and appears not to break bokehjs.